### ISSUE TITLE
Can you share the final loss?

### ISSUE THREAD

**Issue by twmht** [biubug6/Pytorch_Retinaface#40]
Hi,

I am fine tuning the network. Can you share the final loss you trained with mobile0.25?

You may forgot the exact values, but can you give a rough value (like 3.xxx or 4.xxx) ?  I need a reference.

thanks.

**Comment by biubug6** [comment]
sorry, just remember the general range,
loc: about 1.0 , Cla: about 0.X, Landm: about 2.0

**Comment by twmht** [comment]
@biubug6 

that's good for me. thank you.

**Comment by twmht** [comment]
@biubug6 

Did you train for 250 epochs?
